frontend/src/components/dashboard/aliases/images/umbrella-closed.svg

```diff
@@ -1,4 +1,4 @@
-<svg width="54" height="91" viewBox="0 0 54 91" fill="none" xmlns="http://www.w3.org/2000/svg">
+<svg width="104" height="91" viewBox="0 0 54 91" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
You might want to check the source diff here - its width was set to the same width as `umbrella-open.svg` :)